### PR TITLE
Storybook: Add stories for Animate component

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Add stories for Animate component to show in Storybook.

## How has this been tested?

Run: npm run design-system:dev 

Confirm animate component shows in Storybook

### CODE REVIEW

File: packages/components/src/animate/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		) }
+	</Animate>
+);
+
+export const slideIn = () => (
+	<Animate type="slide-in">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Slide-in animation is meant for sidebars and sliding menus.</p>
+				<p>It shows the height and width of the animated element moving from a hidden position to its normal one.</p>
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]: I think that we should just focus on what the animation does. The use cases can be a lot more varied than sidebars and sliding menus.

We could also just avoid descriptions altogether for now until we have more thorough copy. The names are very self descriptive as is.

**Review by epiqueras — CHANGES_REQUESTED** [review]
Thanks for working on this :smile:!

File: packages/components/src/animate/stories/index.js (hunk 1 above)
Comment (mkaz) [reply packages/components/src/animate/stories/index.js]: Thanks for the reviews, I grabbed the content from the README, wasn't quite sure if we should include within.

Comment (epiqueras) [reply packages/components/src/animate/stories/index.js]: What do you think @ItsJonQ?

Comment (ItsJonQ) [reply packages/components/src/animate/stories/index.js]: @epiqueras I think some animation examples would be wonderful 😍 . I would be happy to help here you folks need!

> I grabbed the content from the README
@mkaz Unfortunately, the documentation example doesn't really do a lot 😅

Comment (epiqueras) [reply packages/components/src/animate/stories/index.js]: Yeah, we need better copy, or we can merge the variations without copy for now.

Comment (mkaz) [reply packages/components/src/animate/stories/index.js]: Per this, I've removed the extra documentation and we can improve the README at the same time of what shows for the improved animation examples

File: packages/components/src/animate/stories/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>{ `No default animation. Use one of type = "appear", "slide-in", or "loading".` }</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+// unexported helper for various origins
+const Appear = ( origin ) => (
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
const Appear = ( { origin } ) => (
```

File: packages/components/src/animate/stories/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	>
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Appear animation. Origin: { origin }</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+export const appearTopLeft = () => Appear( 'top left' );
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

File: packages/components/src/animate/stories/index.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+export const appearTopRight = () => Appear( 'top right' );
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

File: packages/components/src/animate/stories/index.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+export const appearBottomLeft = () => Appear( 'bottom left' );
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

File: packages/components/src/animate/stories/index.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+export const appearBottomRight = () => Appear( 'bottom right' );
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
export const appearTopLeft = () => <Appear origin="top left" />
```

File: packages/components/src/animate/stories/index.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+	<Animate>
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>{ `No default animation. Use one of type = "appear", "slide-in", or "loading".` }</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+// unexported helper for various origins
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
// Unexported helper for different origins.
```

File: packages/components/src/animate/stories/index.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+const Appear = ( { origin } ) => (
+	<Animate
+		type="appear"
+		options={ { origin } }
+	>
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Appear animation. Origin: { origin }</p>
```
Comment (epiqueras) [inline packages/components/src/animate/stories/index.js]:
```suggestion
				<p>Appear animation. Origin: { origin }.</p>
```

**Review by epiqueras — APPROVED** [review]

**Pull request merged**